plugins/org.eclipse.fordiac.ide.debug.ui/src/org/eclipse/fordiac/ide/debug/ui/fb/FBLaunchConfigurationTab.java

```diff
@@ -119,7 +213,10 @@ public void initializeFrom(final ILaunchConfiguration configuration) {
 			}
 			repeatEventCheckbox.setSelection(FBLaunchConfigurationAttributes.isRepeatEvent(configuration));
 			keepDebuggerRunningCheckbox
-			.setSelection(FBLaunchConfigurationAttributes.isKeepRunningWhenIdle(configuration));
+					.setSelection(FBLaunchConfigurationAttributes.isKeepRunningWhenIdle(configuration));
+			systemTimeRadio.setSelection(FBLaunchConfigurationAttributes.isSystem(configuration));
+			incrementTimeRadio.setSelection(FBLaunchConfigurationAttributes.isIncrement(configuration));
+			manualTimeRadio.setSelection(FBLaunchConfigurationAttributes.isManual(configuration));
```

Comment:
It seems this does not initialize the time from the configuration, meaning the time is lost when reopening the launch configuration dialog.